sql/moz-fx-data-shared-prod/telemetry_derived/crash_frames_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Crash Frames
+description: |-
+  Unnested frames from crash pings.
+owners:
+- dthorn@mozilla.com
+labels:
+  application: firefox
+  schedule: daily
+  owner: dthorn
```

Comment:
Should we remove this in favour of owner1 label and top level owner key?

Reply:
will do

Reply:
```suggestion
```